microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -73,7 +73,7 @@ Addresses and other form entries are saved by default.  To control saving and au
 
 You can prevent Microsoft Edge from prompting you to save passwords. Navigate to `edge://settings/passwords`. Toggle off the **Offer to save passwords** setting.  You can prevent Microsoft Edge from using autofill for saved passwords and delete saved passwords. Navigate to `edge://settings/passwords` and select **Saved passwords**.  To delete all autofill data, navigate to `edge://settings/clearBrowserData`, select **Autofill form data**, select the desired time range, and then select **Clear now**.
 
-If sync is on for your profile, autofill data is synced across all versions of Microsoft Edge where you are signed in with the same credentials.  When sync is on, all autofill data is stored on encrypted Microsoft servers.  The autofill data stored on Microsoft servers is used only for sync purposes.  You can turn off syncing of your autofill data. Navigate to `edge://settings/profiles/sync`. Select the **Turn off sync** button. If you turned on sync for autofill, deleting autofill data from a device while you are signed in to Microsoft Edge removes the data from all other devices that you are signed in to with the same credentials.
+If you are signed in and syncing, your autofill data is synced across all versions of Microsoft Edge where you are signed in with the same credentials.  When syncing, all autofill data is stored on encrypted Microsoft servers.  The autofill data stored on Microsoft servers is used only for sync purposes.  You can turn off syncing of your autofill data by navigating to `edge://settings/profiles/sync` and deselecting the **Basic info** toggle. If sync is on for autofill, deleting autofill data from a device while you are signed in to Microsoft Edge removes the data from all other devices that you are signed in to with the same credentials.
```

Comment:
@vchapel - Could you please check internal version of Microsoft Writing Style Guide re: using a toggle control?  select, clear, deselect, turn on, turn off, activate, enable, disable, etc.  This is the only occurrence of "deselect" in this article.

Reply:
"turn off" works for me